doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-comparison-legacy-new-ref.adoc

```diff
@@ -0,0 +1,148 @@
+[id='test-scenarios-comparison-legacy-new-ref']
+
+= Feature comparison of legacy and new test scenario designer
+
+{PRODUCT} currently supports both the new *Test Scenarios* designer and the former *Test Scenarios (Legacy)* designer.
+
+The default designer is the new test scenarios designer, which supports testing of both rules and DMN models and provides an enhanced overall user experience with test scenarios. If required, you can continue to use the legacy test scenarios designer, which supports rule-based test scenarios only.
+
+IMPORTANT: The legacy Test Scenarios tool was deprecated with {PRODUCT} 7.3.0. It will be removed in a future {PRODUCT} release. Use the new Test Scenarios designer instead.
+
+The following table highlights the main features of legacy and new test scenario designer which are supported in {PRODUCT} to help you decide the suitable test scenarios designer in your project.
```

Comment:
Suggested rewrite:
The following table highlights the main features of the legacy and new test scenario designer, which are supported in Red Hat Process Automation Manager to help you decide a suitable test scenarios designer in your project.

//adding comma and the

Reply:
Done.